test/templating/react.jsx

```diff
@@ -42,4 +42,4 @@ class Test extends React.Component {
   }
 }
 
-export default translate('react')(Test)
+export default withTranslation('react')(Test)
```

Comment:
Same thing should be done with `typescript.tsx` mock file, but you'll also find that the test is invalid :)